api/v2.go

```diff
@@ -31,16 +31,31 @@ type V2Endpoints struct {
 	AgentAccessTokens   *AgentAccessTokensService
 	Query               *QueryService
 	Policy              *PolicyService
+	Schemas             *SchemasService
 }
 
 func NewV2Endpoints(c *Client) *V2Endpoints {
-	return &V2Endpoints{c,
+	v2 := &V2Endpoints{c,
 		&UserProfileService{c},
 		&AlertChannelsService{c},
 		&CloudAccountsService{c},
 		&ContainerRegistriesService{c},
 		&AgentAccessTokensService{c},
 		&QueryService{c},
 		&PolicyService{c},
+		&SchemasService{c, map[integrationSchema]V2Service{}},
 	}
+
+	v2.Schemas.Services = map[integrationSchema]V2Service{AlertChannels: &AlertChannelsService{c},
+		CloudAccounts: &CloudAccountsService{c}, ContainerRegistries: &ContainerRegistriesService{c}}
```

Comment:
nit: this could be better formatted
```suggestion
	v2.Schemas.Services = map[integrationSchema]V2Service{
		AlertChannels: &AlertChannelsService{c},
		CloudAccounts: &CloudAccountsService{c},
		ContainerRegistries: &ContainerRegistriesService{c},
	}
```